### PR TITLE
MicrosoftCorporation.WindowsPCHealthCheck version 3.0.2109.14001

### PR DESCRIPTION
**Pull request by m4ttyj**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28276)

### CODE REVIEW

File: manifests/m/MicrosoftCorporation/WindowsPCHealthCheck/3.0.2109.14001/MicrosoftCorporation.WindowsPCHealthCheck.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.WindowsPCHealthCheck
```
Comment (jedieaston) [inline manifests/m/MicrosoftCorporation/WindowsPCHealthCheck/3.0.2109.14001/MicrosoftCorporation.WindowsPCHealthCheck.installer.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.WindowsPCHealthCheck

```

File: manifests/m/MicrosoftCorporation/WindowsPCHealthCheck/3.0.2109.14001/MicrosoftCorporation.WindowsPCHealthCheck.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.WindowsPCHealthCheck
```
Comment (jedieaston) [inline manifests/m/MicrosoftCorporation/WindowsPCHealthCheck/3.0.2109.14001/MicrosoftCorporation.WindowsPCHealthCheck.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.WindowsPCHealthCheck

```

File: manifests/m/MicrosoftCorporation/WindowsPCHealthCheck/3.0.2109.14001/MicrosoftCorporation.WindowsPCHealthCheck.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.WindowsPCHealthCheck
```
Comment (jedieaston) [inline manifests/m/MicrosoftCorporation/WindowsPCHealthCheck/3.0.2109.14001/MicrosoftCorporation.WindowsPCHealthCheck.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.WindowsPCHealthCheck

```

**Review by jedieaston — CHANGES_REQUESTED** [review]
wingetcreate tries to put Microsoft software in "MicrosoftCorporation", but we use "Microsoft" instead. Once you apply these changes, you'll also have to move it to the Microsoft folder. 

Thanks!

**Review by jedieaston — CHANGES_REQUESTED** [review]
Hot tip to change the folder name: Press "." on your keyboard when you have GitHub open to this PR in your browser, and a web version of VSCode will launch. 

Once it loads, find the files you added, and drag the WIndowsPCHealthCheck folder to Microsoft. Then use the source control button on the left to commit (it's automatically pushed).

image

**Comment by Rolling2405** [comment]
@m4ttyj  I already have a pull request with this manifest [https://github.com/microsoft/winget-pkgs/pull/28359](url)

**Comment by Rolling2405** [comment]
#28359

**Pull request closed without merging**